documentation/adoc/cluster-controller.adoc

```diff
@@ -337,3 +337,6 @@ The controller itself can be configured through the following environment variab
 Default: `strimzi.io/kind=cluster`.  
 * `STRIMZI_FULL_RECONCILIATION_INTERVAL` : the interval between periodic reconciliations.
 Default: 120000 ms
+* `STRIMZI_OPERATION_TIMEOUT` : Timeout for internal operations specified in milliseconds. This value should be
```

Comment:
Should we make clear that this timeout applies separately to each operation (SS readiness, Pod readiness, Service readiness) and not to the whole logical operation that's made up of those individual operations?

Reply:
I don't think we should get into details. Or at least I'm not sure how to write it without spending too much time on describing hot the Cluster Controller works. I think the current level is sufficient.